test-cases/performance/perf-regression-throughput-125gb-with-ebs-gp3-volume.yaml

```diff
@@ -0,0 +1,15 @@
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=ALL n=125000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=100 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1..125000000"
+stress_cmd_r: "cassandra-stress read no-warmup  cl=QUORUM duration=60m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=90' -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(1..125000000,62500000,125000000)' "
+stress_cmd_m: "cassandra-stress mixed no-warmup cl=QUORUM duration=60m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=90' -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(1..125000000,62500000,125000000)' "
+
+instance_type_db: 'r5b.2xlarge'
```

Comment:
Do we must use r5b or gp3?
If so, can we use also r5b in the other gp2 tests?

Reply:
I see you changed it, don't know if tested it, but for now it's ok.
We shall follow with more tests.